Cargo.lock

```diff
@@ -202,35 +196,53 @@ version = "1.8.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "9ac0150caa2ae65ca5bd83f25c7de183dea78d4d366469f148435e2acfbad0da"
 
+[[package]]
+name = "cairo-annotations"
+version = "0.2.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "e8b023ba5284998fe3a8c9e0374fd1ca51f61da08c5c7f909c828b349fb18143"
+dependencies = [
+ "cairo-lang-sierra",
+ "cairo-lang-sierra-to-casm",
+ "camino",
+ "derive_more",
+ "regex",
+ "serde",
+ "serde_json",
+ "strum",
+ "strum_macros",
+ "thiserror",
+]
+
 [[package]]
 name = "cairo-lang-casm"
-version = "2.7.0"
+version = "2.8.5"
```

Comment:
Did you run `cargo update`? It is not a good practice to do it randomly, sth make break. I would revert and just bump cairo deps to 2.8.5 in toml

Reply:
Hmm, this probably comes from cairo-annotations as it's the dependency there. Will add to the toml.
Btw what was the reason to have it set to 2.7? was it forge?

Reply:
updated in https://github.com/software-mansion/cairo-profiler/pull/122/commits/1ec7a8c719f59800b1f2afac66c40cfb217705f4